packages/core/src/utils/files.ts

```diff
@@ -53,14 +58,20 @@ export const writePackageJson = async <T extends PackageJson>({
 
 export const emptyDir = async (dir: string, skip = ['.git']): Promise<void> => {
   if (!existsSync(dir)) {
-    fs.mkdir(dir, { recursive: true });
+    await fs.mkdir(dir, { recursive: true });
     return;
   }
 
   for (const file of await fs.readdir(dir)) {
-    if (skip?.includes(file)) {
+    if (skip.includes(file)) {
       continue;
     }
     await fs.rm(path.resolve(dir, file), { recursive: true, force: true });
   }
 };
+
+export const extensionForFormat = (format: Format) =>
+  (({ esm: 'esm.js', cjs: 'cjs', dts: 'cjs.d.ts' } as const)[format]);
+
+export const toRollupFormat = (format: Format): ModuleFormat =>
+  (({ esm: 'esm', cjs: 'cjs', dts: 'esm' } as const)[format]);
```

Comment:
Nitpick: these don't really need to be functions, they'd work just as well as typed objects: